src/Tests/dotnet-new.Tests/Approvals/DotnetNew3CompleteTests.CanDoTabCompletion.Linux.verified.txt

```diff
@@ -1,20 +0,0 @@
-﻿classlib
```

Comment:
I see 3 files after my changes:
src/Tests/dotnet-new.Tests/Approvals/DotnetNewCompleteTests.CanDoTabCompletion.Linux.verified.txt

src/Tests/dotnet-new.Tests/Approvals/DotnetNewCompleteTests.CanDoTabCompletion.OSX.verified.txt

src/Tests/dotnet-new.Tests/Approvals/DotnetNewCompleteTests.CanDoTabCompletion.Windows.verified.txt

It might be that Git didn't detect the rename changes correctly.

Reply:
I guess I know what happened.

Previous version of Linux file was not correct - it was missing a lot of options, Win and OSX was correct. 

My changes fixed Linux test. 

Therefore
- Windows file is shown as renamed - there were no changes 
- Git detected OSX as rename to Linux (wrongly but ok)
- Linux file was removed. OSX file was created. This is not shown as rename as the changes are more that 10%.

Reply:
What I mean by missing file was rather "missed content of the file". If the content was wrong and it was not just a straightforward rename as it seemed to me, but a change of the Linux file, then everything is fine.